html-forms/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> html-forms </title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form action="http://www.example.org/subscribe.php">
+      <div>
+        Email
+      </div>
+      <input type="email" name="email" />
+    </form>
```

Comment:
This is an extra closing tag, you already have a closing tag for the `form` after all the form controls on line 75. Your indentation got really messed up in this file, this is probably why. Please fix your indentation as well.